### PR TITLE
CSS cleanup 

### PR DESCRIPTION
**Pull request by Vbeany**
Added comments to all sections of CSS files to better explain what element and class it belongs to on BloomingLeaf and cleaned up unneeded comments. Also removed dead code.

### CODE REVIEW

File: leaf-ui/css/analysis_html.css [hunk 1]
```diff
@@ -132,42 +147,44 @@ body {
...
 {
 	border-radius: 10px;
   height: 20px;
 	background-color: #AAAAAA;
 }
 
+/* Sets attributes for the Dropdown Satisfaction Menu */
 #sat-value {
-  background-color: rgba(36, 150, 255, 1) !important;
+  background-color: rgb(36, 150, 255, 1) !important;
```
Comment (i-m-foster) [inline leaf-ui/css/analysis_html.css]: Why'd you delete the a here but not in the other rgba places? I think you should be able to also delete the "a" and the "1" in all of those places as long as the fourth number is a 1

File: leaf-ui/css/style.css [hunk 2]
```diff
@@ -50,7 +48,7 @@ body {
   font-style: italic;
 }
 
-/** Dims the entire page **/ 
+/** Dims the entire page when behind the"Create My Palette" popup**/
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Small comment but add a space between `the` and the `"`

File: leaf-ui/css/style.css [hunk 3]
```diff
@@ -61,144 +59,120 @@ body {
...
+/* Sets Margin distance for the right panel */
+#right-inspector {
   margin-top: 50px;
 }
 
+/* Sets attributes for input text boxes on the right panel */
+.inspector input, .inspector textarea {
+  color: black;
+  text-shadow: none;
+  /* box-shadow: inset 0 0 0 1px rgba(0,0,0,0.1),0px 0 0 1px rgba(255,255,255,0.1); */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Can this be deleted?

File: leaf-ui/css/style.css [hunk 4]
```diff
@@ -215,89 +190,54 @@ body {
...
-  padding: 0;
-  color: white;
-  text-align: center;
-  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.25);
-  border: 0;
-  cursor: pointer;
-  margin-left: 5px;
-}
-
+/* Attributes for "Num Relative TIme Point" input box on left panel on Analysis mode */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Small comment - TIme to Time

File: leaf-ui/css/style.css [hunk 5]
```diff
@@ -348,43 +291,50 @@ body {
...
 }
+
+/* Sets attributes for error messages on the right panel "repeat-error" */
 #repeat-end2, #repeat-end3 {
   width: 101px;
   margin-top: 4px;
   margin-bottom: 8px;
   border-radius: 5px;
 }
+
```
Comment (YesugenB) [inline leaf-ui/css/style.css:333]: Delete extra space

File: leaf-ui/css/style.css [hunk 6]
```diff
@@ -446,9 +398,28 @@ select > .select-placeholder {
...
+  float: left;
+  font-size: 1em;
+  font-weight: lighter;
+}
+
+#clear-query-btn {
+  margin: 0;
+}
+
+#query-btn {
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Maybe add comment here? I'm not sure what query-btn is

File: leaf-ui/css/style.css [hunk 7]
```diff
@@ -460,66 +431,33 @@ select > .select-placeholder {
...
-}
-
+/* Any child of this class makes btns blue, ex. "Add", "Set Repeats" btn on right panel */
 .blue-btn {
   background-color: rgba(36, 150, 255, 1) !important;
   -webkit-box-shadow: inset 0 -2px rgba(29, 122, 208, 1);
   box-shadow: inset 0 -2px rgba(29, 122, 208, 1);
 }
 
+/* Any child of this class makes btns blue, ex."Clear" btn on right panel */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: red-btn makes buttons blue?

File: leaf-ui/css/style.css [hunk 8]
```diff
@@ -528,53 +466,44 @@ select > .select-placeholder {
...
 }
 
-/* beggining of representative color slider */
+/* Beginning of representative color slider */
 
 .slidecontainer {
   width: 100%; /* Width of the outside container */
 }
+
+/* Sets attributes for text labels on the toolbar on modeling mode, no text labels are using this id.. */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: If no text labels are using this id, can this be deleted? Same for `#top-label-analysis`

File: leaf-ui/css/style.css [hunk 9]
```diff
@@ -610,15 +541,21 @@ select > .select-placeholder {
   margin-bottom: 0px;
   margin-left: 10px;
   }
+
+  /* Sets position of EVO slider */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: Move indent to the left

File: leaf-ui/css/style.css [hunk 10]
```diff
@@ -677,8 +620,7 @@ select > .select-placeholder {
 
 /* end of representive slider styling */
 
-/* end of representive slider styling */
-
+/* Sets atrributes for all children of btn class including "File, Edit, View, Help, and etc.."  */
```
Comment (YesugenB) [inline leaf-ui/css/style.css]: atrributes -> attributes I think?

File: leaf-ui/css/analysis_style.css [hunk 11]
```diff
@@ -278,19 +289,20 @@
     background: #f8f8f8;
     width: 96px;
     -webkit-appearance:none; /* remove the strong OSX influence from Webkit */
-
     -webkit-border-radius: 6px;
     -moz-border-radius: 6px;
     border-radius: 6px;
 }
+
 #relationshipList:focus{
```
Comment (i-m-foster) [inline leaf-ui/css/analysis_style.css:298]: what does the :focus mean? it might be good to have a comment here

File: leaf-ui/css/analysis_style.css [hunk 12]
```diff
@@ -380,12 +393,14 @@ select::-ms-expand {
     float: left;
     }
 
+/* Sets attributes and look of dropdown btns carrot symbol on left panel on Analysis mode */
```
Comment (i-m-foster) [inline leaf-ui/css/analysis_style.css]: i think its spelled caret in this case

File: leaf-ui/css/style.css [hunk 13]
```diff
@@ -348,43 +291,50 @@ body {
...
+/* Sets attributes for error messages on the right panel "repeat-error" */
 #repeat-end2, #repeat-end3 {
   width: 101px;
   margin-top: 4px;
   margin-bottom: 8px;
   border-radius: 5px;
 }
+
+
+/* This section deals with the repeat section when function type on right panel is set to userdefine*/
```
Comment (i-m-foster) [inline leaf-ui/css/style.css]: i think userdefine should be "User defined"

File: leaf-ui/css/style.css [hunk 14]
```diff
@@ -446,9 +398,28 @@ select > .select-placeholder {
...
+
+#clear-query-btn {
+  margin: 0;
+}
+
+#query-btn {
+  margin: 16px 0;
+}
+
+/* This section ends the repeat section when function type on right panel is set to userdefine */
```
Comment (i-m-foster) [inline leaf-ui/css/style.css]: i think this should also be "User defined"

File: leaf-ui/css/analysis_style.css [hunk 15]
```diff
@@ -397,22 +412,29 @@ select::-ms-expand {
...
 }
 
-#removeConstraint{
+/* Changes cursor to pointer when removing a constraint on "View Assignemnt List modal" */
+#removeConstraint {
     cursor: pointer;
 }
 
-#abs-time-pts{
+/* Sets margin for "Absolute Time Points" text input box on "Absolute and Relative ASsignments modal" */
```
Comment (YesugenB) [inline leaf-ui/css/analysis_style.css]: ASsignments to Assignments

File: leaf-ui/css/style.css [hunk 16]
```diff
@@ -446,9 +398,28 @@ select > .select-placeholder {
...
+
+
+/* Sets attributes for any child element of inspector-btn; currently there are no such elements*/
```
Comment (i-m-foster) [inline leaf-ui/css/style.css]: if there are no child elements can we delete this?

File: leaf-ui/css/style.css [hunk 17]
```diff
@@ -732,13 +674,15 @@ select > .select-placeholder {
...
 
+/* Sets attributes for each btn on dropdowns on the toolbar */
 .dropdown-toolbar a {
   color: black;
   padding: 12px 16px;
   text-decoration: none;
   display: block;
 }
 
+/* When btn on dropdown menus are hovered above,  bg color is darkened */
```
Comment (i-m-foster) [inline leaf-ui/css/style.css]: really minor but fix the spacing before bg. also does bg stand for background?

File: leaf-ui/css/analysis_html.css [hunk 18]
```diff
@@ -253,29 +272,26 @@ body {
   background: #424242;
 }
 
-/*Click effect */
+/* When trash btn is click on click effect */
```
Comment (YesugenB) [inline leaf-ui/css/analysis_html.css]: Maybe reword? "When trash btn is clicked.."

File: leaf-ui/css/analysis_html.css [hunk 19]
```diff
@@ -253,29 +272,26 @@ body {
...
 .table-btn-small:active {
   background-color: #424242;
   box-shadow: inset 0 -2px #292929;
   transform: translateY(2px);
 }
 
-.blue-btn{
+/* Sets attributes for blue btn is set, Filter apply btn is a child of blue-btn */
```
Comment (YesugenB) [inline leaf-ui/css/analysis_html.css]: Either "Sets attributes for blue btn" or "attributes for blue btn is set" ?

**Review by i-m-foster — APPROVED** [review]
looks good to me! just a few minor questions

**Review by YesugenB — APPROVED** [review]
Looks good! I left some small comments
